shared/cfg/guest-hw.cfg

```diff
@@ -70,8 +77,10 @@ variants:
         drive_format=virtio
         i440fx:
             cd_format=ide
-        q35:
+        q35, q35-with-bridge:
```

Comment:
Oh I didn't notice it'd sprawl so extensively, perhaps I was being to careful. I'm wondering what the q35 would look like in libvirt, do you have any news whether they're going to do similar setup? If (and only if) they are then we can probably just redefine the q35 and avoid the many new branches.